SharedUtils/src/main/java/com/braintreepayments/api/sharedutils/Time.java

```diff
@@ -0,0 +1,11 @@
+package com.braintreepayments.api.sharedutils;
+
+import androidx.annotation.RestrictTo;
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+public class Time {
```

Comment:
Nit: any reason why we can't make this a Kotlin class?

Reply:
Good call! I initially thought the shared utils module was all Java. I'll refactor and push up the change.